ci-operator/step-registry/firewatch/report-issues/firewatch-report-issues-commands.sh

```
@@ -42,6 +42,10 @@ if [ -f "${SHARED_DIR}/${FIREWATCH_JIRA_ADDITIONAL_LABELS_FILE}" ]; then
     report_command+=" --additional-labels-file=${SHARED_DIR}/${FIREWATCH_JIRA_ADDITIONAL_LABELS_FILE}"
 fi
 
+if [ -f /tmp/secrets/slack/url ]; then
+    export SLACK_WEBHOOK_URL=$(cat /tmp/secrets/slack/url)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)